holoviews/core/data/multipath.py

```diff
@@ -20,7 +20,9 @@ class MultiInterface(Interface):
 
     datatype = 'multitabular'
 
-    subtypes = ['dataframe', 'dictionary', 'array', 'dask']
+    subtypes = ['dictionary', 'dataframe', 'array', 'dask']
```

Comment:
I *think* it makes sense to try the more general dictionary (i.e standard python literals) format first. Might be other implications I haven't figured out yet. Then again, ``MultiInterface`` is pretty new so it probably doesn't matter wrt backwards compatibility.

Reply:
Right, the point here is that it doesn't accidentally expand your scalar values into a dataframe column for instance.

Reply:
Makes sense.